### PR TITLE
EZ-350: Article Autosave

### PR DESCRIPTION
**Pull request by saurabh-2k17**

### CODE REVIEW

File: modules/ezcontent_node/modules/ezcontent_landing_page/config/install/core.entity_view_display.node.landing_page.default.yml [hunk 1]
```diff
@@ -77,6 +77,22 @@ third_party_settings:
...
+          - 'field_block:node:landing_page:title'
+        Help: {  }
+        Menus: {  }
+        'Moderation Dashboard': {  }
+        'Responsive Preview': {  }
+        System: {  }
+        User: {  }
+        core: {  }
+      blacklisted_blocks: {  }
+    allowed_block_categories: {  }
```
Comment (vaibhavjain-in) [inline modules/ezcontent_node/modules/ezcontent_landing_page/config/install/core.entity_view_display.node.landing_page.default.yml]: Is this config related to Autosave ?

**Review by vaibhavjain-in — CHANGES_REQUESTED** [review]
@saurabh-2k17 Your commit history is not clean, can you please see.
Also cater to comment.

File: modules/ezcontent_node/modules/ezcontent_landing_page/config/install/core.entity_view_display.node.landing_page.default.yml (hunk 1 above)
Comment (saurabh-2k17) [reply modules/ezcontent_node/modules/ezcontent_landing_page/config/install/core.entity_view_display.node.landing_page.default.yml]: No, i have also pushed EZ-195 in the same branch. I have updated the message for more clarity

**Review by vaibhavjain-in — CHANGES_REQUESTED** [review]
@saurabh-2k17 I believe, we missed adding this to composer.json
Also, we missed adding the config, where we enable this for Article Content Type.

**Review by vaibhavjain-in — APPROVED** [review]
LGTM
